account_move_base_import/models/account_journal.py

```diff
@@ -0,0 +1,287 @@
+                    _('No commission account is set on the journal.'))
+            else:
+                commission_account_id = self.commission_account_id.id
+                comm_values = {
+                    'name': _('Commission line'),
+                    'date_maturity': parser.get_move_vals().get('date') or
+                    fields.Date.today(),
+                    'debit': -global_commission_amount,
+                    'partner_id': partner_id,
+                    'move_id': move.id,
+                    'account_id': commission_account_id,
+                    'already_completed': True,
+                }
+                move_line_obj.with_context(
+                    check_move_validity=False
+                ).create(comm_values)
+
+        # Counterpart line
+        if total_amount > 0.0:
+            account_id = self.default_debit_account_id.id
+            credit = 0.0
+            debit = total_amount
+        else:
+            account_id = self.default_credit_account_id.id
+            credit = -total_amount
+            debit = 0.0
+        counterpart_values = {
+            'name': _('/'),
+            'date_maturity': parser.get_move_vals().get('date') or
+            fields.Date.today(),
+            'credit': credit,
+            'debit': debit,
+            'partner_id': partner_id,
+            'move_id': move.id,
+            'account_id': account_id,
+            'already_completed': True,
+        }
+        move_line_obj.create(counterpart_values)
+
+    @api.multi
+    def write_logs_after_import(self, move, num_lines):
+        """Write the log in the logger
+
+        :param int/long statement_id: ID of the concerned
+          account.bank.statement
+        :param int/long num_lines: Number of line that have been parsed
+        :return: True
+        """
+        self.message_post(
+            body=_('Move %s have been imported with %s '
+                   'lines.') % (move.name, num_lines))
+        return True
+
+    def prepare_move_line_vals(self, parser_vals, move):
+        """Hook to build the values of a line from the parser returned values.
+        At least it fullfill the basic values. Overide it to add your own
+        completion if needed.
+
+        :param dict of vals from parser for account.bank.statement.line
+          (called by parser.get_st_line_vals)
+        :param int/long statement_id: ID of the concerned
+          account.bank.statement
+        :return: dict of vals that will be passed to create method of
+          statement line.
+        """
+        move_line_obj = self.env['account.move.line']
+        values = parser_vals
+        values['company_id'] = self.company_id.id
+        values['currency_id'] = self.currency_id.id
+        values['company_currency_id'] = self.company_id.currency_id.id
+        values['journal_id'] = self.id
+        values['move_id'] = move.id
+        values['account_id'] = self.receivable_account_id.id
```

Comment:
In the generic parser, there is a part adding account_id in the parser_vals, but the data is lost anyway here no?